src/components/home/home.styles.js

```diff
@@ -73,16 +73,16 @@ const useHomeStyles = createUseStyles(theme => ({
     margin: '24px 0'
   },
   section4: {
-    '& >div': {
-      paddingTop: '48px'
+    '& $divider': {
```

Comment:
A general comment for this file, I would create dedicated classes instead of targeting nested elements in the styles, I think it's more clear and easy to read and understand after some time without touching a project.

Reply:
I would argue that it is clear like this. I mean there are some shared elements, like this `divider` that need to get particular styles for different sections.
For me it is clear to have `section4` and then nested `& $divider`, rather than adding a new class for each section (for example `dividerSection4`. But if you guys prefer it like that I'll make changes.